src/main/java/uk/gov/companieshouse/acsp/manage/users/configuration/MongoRoleReadingConverter.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.acsp.manage.users.configuration;
```

Comment:
Unit tests are probably easy to make for these too

Reply:
Added tests

Reply:
Resolved code smells